automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -0,0 +1,199 @@
+package org.broadinstitute.dsde.firecloud.test.security
+
+import org.broadinstitute.dsde.firecloud.api.{AclEntry, WorkspaceAccessLevel}
+import org.broadinstitute.dsde.firecloud.config.{AuthToken, AuthTokens, Config}
+import org.broadinstitute.dsde.firecloud.fixture.{GroupFixtures, WorkspaceFixtures}
+import org.broadinstitute.dsde.firecloud.page.workspaces.WorkspaceSummaryPage
+import org.broadinstitute.dsde.firecloud.test.{CleanUp, WebBrowserSpec}
+import org.scalatest._
+
+/*
+ * This test SHOULD be able to run with ParallelTestExecution. However, Rawls
+ * currently has database deadlock problems when creating and deleting managed
+ * groups because they involve 3 operations across 2 tables. My initial
+ * attempt to retry the deadlock failed because Rawls also creates the groups
```

Comment:
Interesting that this causes Rawls deadlocks - I wonder if this is a case of https://broadinstitute.atlassian.net/browse/GAWB-2256..